waku/v2/node/wakunode2.nim

```diff
@@ -1168,9 +1172,13 @@ when isMainModule:
     ## Start a configured node and all mounted protocols.
     ## Resume history, connect to static nodes and start
     ## keep-alive, if configured.
-
-   # Start Waku v2 node
+    
+    # Start Waku v2 node
     waitFor node.start()
+
+    # start discv5 and connect to discovered nodes
+    if not waitFor node.startDiscv5():
```

Comment:
This will attempt to start discv5, even if it's disabled. Even though this will fail quickly, it does trigger code which is unexpected. Perhaps only call this if `conf.discv5Discovery`?

Reply:
done. 😅